graylog2-server/src/main/java/org/graylog2/rest/resources/system/SessionsResource.java

```diff
@@ -88,6 +88,8 @@
     private final SessionResponseFactory sessionResponseFactory;
     private final ClusterConfigService clusterConfigService;
 
+    private static final String USERNAME = "username";
```

Comment:
we should be using that constant also when reading the username.
e.g:
https://github.com/Graylog2/graylog2-server/blob/4.2/graylog2-server/src/main/java/org/graylog2/shared/security/ActorAwareUsernamePasswordTokenFactory.java#L29

Reply:
This touches code not related to the security fix. Let's postpone that.